notification/rule/http.go

```diff
@@ -0,0 +1,154 @@
+package rule
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/notification"
+	"github.com/influxdata/influxdb/notification/flux"
+)
+
+// HTTP is the notification rule config of http.
+type HTTP struct {
+	Base
+	URL string
+}
+
+// GenerateFlux generates a flux script for the http notification rule.
+func (s *HTTP) GenerateFlux(e influxdb.NotificationEndpoint) (string, error) {
+	// TODO(desa): still needs full implementation
```

Comment:
what's missing here? (just curious)

Reply:
The flux we need to generate needs to match the flux here https://github.com/influxdata/idpe/pull/4342#pullrequestreview-276100524

I've had to to some coding around the fact that the alerts package isn't available in flux yet. Hence the jankiness here.